sdk/core/Azure.Core/tests/ModelSerializationTests/Animal.cs

```diff
@@ -0,0 +1,164 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text.Json;
+
+namespace Azure.Core.Tests.ModelSerializationTests
+{
+    public class Animal : IJsonSerializable, IUtf8JsonSerializable
+    {
+        private Dictionary<string, BinaryData> RawData { get; set; } = new Dictionary<string, BinaryData>();
```

Comment:
Should this be "RawData" or "RawLeftOvers"?
RawData reads like it contains everything but from the implementation below, it actually only contains left overs

Reply:
Maybe `RawAdditionalProperties`?